pages/k8s/certs-and-trust.md

```diff
@@ -0,0 +1,247 @@
+---
+wrapper_template: "base-docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Certificates and Trust Management"
+  description: How CDK manages PKI certificates and trust
+keywords: juju, kubernetes, cdk, security, pki, encryption, vault, ha
+tags: [operating, security]
+sidebar: k8smain-sidebar
+permalink: certs-and-trust.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+# Securing communications with PKI
```

Comment:
This part of the document starts off a (very good!) reference of PKI and certificates